src/Tribe/Integrations/WPML/WPML.php

```diff
@@ -80,6 +84,40 @@ protected function hook_filters() {
 			$category_translation = Tribe__Events__Integrations__WPML__Category_Translation::instance();
 			add_filter( 'tribe_events_category_slug', array( $category_translation, 'filter_tribe_events_category_slug' ), 20, 2 );
 		}
+
+		//// WORK IN PROGRESS!!!
+
+		add_action( "tribe_repository_events_pre_get_posts", function () {
+			$request_uri = isset( $_SERVER['REQUEST_URI'] )
+				? filter_var( $_SERVER['REQUEST_URI'], FILTER_SANITIZE_STRING )
+				: false;
+			if(false === $request_uri){
+				return;
+			}
+			$url_language_code = false;
+			global $sitepress;
+			$languages               = array_column( (array) $sitepress->get_active_languages(), 'default_locale' );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 12 spaces but found 15 spaces

<sub>hash: fcd35868d22739ffb4671d72f4969894</sub>